src/Components/Home/Home.jsx

```diff
@@ -1,7 +1,9 @@
 import React, { Fragment } from "react";
+import AuthSidebar from "../Shared/AuthSidebar/AuthSidebar.jsx";
 
 const Home = () => (
   <Fragment>
+    <AuthSidebar active="signUp" />
     <section className="main">
       <h2>Home Page</h2>
```

Comment:
this `h1` element is appearing in the signup, I would advice to remove it since it has not been worked on yet.
![image](https://user-images.githubusercontent.com/11549858/82920897-2bebff80-9f78-11ea-8b28-ba79d0d5518a.png)

Reply:
Yeah, Feedback Implemented